apps/prairielearn/src/cron/autoFinishExams.ts

```diff
@@ -16,29 +19,33 @@ import { config } from '../lib/config';
  * @see assessment.gradeAssessmentInstance
  */
 export async function run() {
-  const params = [config.autoFinishAgeMins];
-  const result = await sqldb.callAsync('assessment_instances_select_for_auto_finish', params);
-  const examList = result.rows;
+  const examList = await sqldb.queryRows(
```

Comment:
Nit: as long as we're refactoring, call this `assessments`?

Reply:
Renamed to assessment_instances in f04ef40.